### ISSUE TITLE
Home Page: "Get Started" CTA color contrast issue

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3166]
On the home page, the "Get Started" CTA is failing some color contrast checks- the white on the ember orange background is failing: 
![image](https://user-images.githubusercontent.com/4587451/35195551-dd048a58-fe8a-11e7-81d8-506cff9ef425.png)

There are a few options to fix: 
1. increase the font size to 18.66px and make the text bold: 
![image](https://user-images.githubusercontent.com/4587451/35195565-0c52ac2c-fe8b-11e7-8251-4307e3a95a69.png)

2. increase the font size to 24px:
![image](https://user-images.githubusercontent.com/4587451/35195577-3baa54ca-fe8b-11e7-89e7-01c73e7bfcc2.png)

3. come up with a different color combination for the bottom of the CTA. 

FWIW, if any text is on Ember Orange, it will require this consideration, so it may be efficient to decide on a desired approach in general. Here's more context: 
![image](https://user-images.githubusercontent.com/4587451/35195607-bdc9127a-fe8b-11e7-8df2-1d039d010e88.png)

**Comment by locks** [comment]
I am partial to option 1 for this specific case, but not sure as a systematic solution.

**Comment by svinkle** [comment]
Option 3 seems the most robust. It guarantees any size/weight "white on orange"  text will pass AA.

That said, if the orange branding can't be modified, option 1 might be best.

**Comment by cdvillard** [comment]
I'm between options 1 and 2, and more towards no. 2 to keep the visual hierarchy from getting confusing.

**Comment by stringyland** [comment]
I like option 1 for this situation, since it’s a starting point and large text is always useful there. But as a general solution option 3 will very likely be easier to implement in a variety of cases.
